client/src/pages/Availability/useStyles.ts

```diff
@@ -0,0 +1,25 @@
+import { makeStyles } from '@material-ui/core/styles';
+import { theme } from '../../themes/theme';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    maxWidth: '800px',
+    margin: `${theme.spacing(4)}px auto`,
+    padding: theme.spacing(4),
+    alignSelf: 'center',
+  },
+  form: {
```

Comment:
These properties can be applied to the `Box` component directly, please see comment above